.github/workflows/pr.yml

```diff
@@ -27,28 +35,37 @@ jobs:
 
       - name: Render Book
         run: Rscript -e 'bookdown::render_book("index.Rmd")'
-      - name: If this is a fork stop here
-        run: |
-          fork=$(jq --raw-output .pull_request.head.repo.fork "${GITHUB_EVENT_PATH}")
-          if [ $fork == true ]; then
-             echo "This is a fork, no preview rendering"
-             exit 78;
-          else
-             echo "This is not a fork, on to preview rendering"
-          fi
+        
       - uses: actions/setup-node@v1
+        if: env.fork == 'false'
         with:
           node-version: "12.x"
+          
       - name: Install Netlify CLI
+        if: env.fork == 'false'
         run: npm install netlify-cli -g
+        
       - name: Deploy to Netlify (test)
+        if: env.fork == 'false'
         run: DEPLOY_URL=$(netlify deploy --site ${{ secrets.NETLIFY_SITE_ID }} --auth ${{ secrets.NETLIFY_AUTH_TOKEN }} --dir=docs --json | jq '.deploy_url' --raw-output);echo "::set-env name=DEPLOY_URL::$DEPLOY_URL"
-      - name: Create commit comment
-        uses: peter-evans/commit-comment@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          sha: ${{ github.event.pull_request.head.sha }}
-          body: |
-            You can go look at [the preview][1]. :bike: :boat:
-
-            [1]: ${process.env.DEPLOY_URL}
+                
+      - name: Create check
+        if: env.fork == 'false'
+        run: |
+          curl --request POST \
+          --url https://api.github.com/repos/${{ github.repository }}/check-runs \
+          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+          --header 'Accept: application/vnd.github.antiope-preview+json' \
+          --header 'content-type: application/json' \
+          --data '{
+            "name": "Preview Book",
+            "external_id": "42",
+            "head_sha": "${{ github.event.pull_request.head.sha }}",
+            "conclusion": "success",
+            "html_url": "${{ env.DEPLOY_URL }}",
+            "details_url": "${{ env.DEPLOY_URL }}",
```

Comment:
this is ignored by GitHub. see https://github.com/ropenscilabs/statistical-software-peer-review/pull/7/checks?check_run_id=467011737#step:13:1